airflow/ui/package.json

```diff
@@ -17,18 +17,20 @@
   },
   "dependencies": {
     "@chakra-ui/anatomy": "^2.2.2",
-    "@chakra-ui/react": "^3.0.2",
+    "@chakra-ui/react": "^3.1.1",
     "@codemirror/lang-json": "^6.0.1",
     "@emotion/react": "^11.13.3",
     "@tanstack/react-query": "^5.52.1",
     "@tanstack/react-table": "^8.20.1",
-    "@uiw/react-codemirror": "^4.23.5",
     "@uiw/codemirror-themes-all": "^4.23.5",
+    "@uiw/react-codemirror": "^4.23.5",
     "axios": "^1.7.7",
     "chakra-react-select": "6.0.0-next.2",
+    "chart.js": "^4.4.6",
```

Comment:
Just curious with chartjs since legacy UI used echarts for many charts. On first look the chartjs API looks similar to echarts. Were there any issues with echarts that led to this decision?

Reply:
If chart.js has everything that we need in terms of variety of charts and customization option, indeed I think that would be simpler to use. 👍.

> Just curious with chartjs since legacy UI used echarts for many charts. On first look the chartjs API looks similar to echarts. Were there any issues with echarts that led to this decision?

I don't think so, it's just that `chart.js` has really good defaults and is really easy to use in comparison I think.

Reply:
I tried using echarts for this and it was very difficult to size and position it properly and without typescript errors. And chartjs has better documentation.